### ISSUE TITLE
Device or resource busy

### ISSUE THREAD

**Issue by manodupont** [flavienbwk/opensearch-docker-compose#5]
Hi, i found this stack which looks pretty good.

But as soon as I launched i i always have that kind of error 
```
[opensearch1]$ docker-compose up -d os01 && dl os01
[+] Running 2/2
 ⠿ Network opensearch1_default   Created                                                                                                                                         0.1s
 ⠿ Container opensearch1-os01-1  Started                                                                                                                                         1.1s
opensearch1-os01-1  | Disabling execution of install_demo_configuration.sh for OpenSearch Security Plugin
opensearch1-os01-1  | Enabling OpenSearch Security Plugin
opensearch1-os01-1  | sed: cannot rename /usr/share/opensearch/config/sedQOWovq: Device or resource busy
^Ccanceled
[opensearch1]$ ^C
```

I have tried many combinations of users: groups, mod, etc on the certs folder and still it doesnt work.

Any idea ?

Thanks

**Comment by flavienbwk** [comment]
Hi @manodupont, thanks for your feedback.

I can reproduce the issue : 

```
os03_1    | sed: cannot rename /usr/share/opensearch/config/sedRvwQ9h: Device or resource busy
os02_1    | sed: cannot rename /usr/share/opensearch/config/sed1mJR9h: Device or resource busy
os01_1    | sed: cannot rename /usr/share/opensearch/config/sedQnlZUf: Device or resource busy
```

Looks like a bug inside `opensearchproject/opensearch` image. This is still a young fork.

Does this causes you any trouble ?

**Comment by manodupont** [comment]
Well, yes! ahaha i was hoping to find a solution. So do you think the 1.0.1 version might be better suited ?! Actually I was wondering how you made that work cause i suppose that if you put this stack on github, it was because it was working, right ?

or maybe not ? :)

**Comment by flavienbwk** [comment]
Could you be more explicit about what is not working ?
I am able to login into Kibana, inject and visualize data. Working on Ubuntu.

**Comment by flavienbwk** [comment]
This looks like fixed in [`1.2.4`](https://github.com/opensearch-project/opensearch-build/pull/1130)
